main/field.c

```diff
@@ -229,12 +229,15 @@ static const char *renderEscapedName (const char* s,
 		int c = *s;
 		if ((c > 0x00 && c <= 0x1F) || c == 0x7F)
 		{
-			notice("Unexpected character (0 < *c && *c < 0x20) included in a tagEntryInfo: %s", base);
-			notice("File: %s, Line: %lu, Lang: %s, Kind: %c",
-			       tag->inputFileName, tag->lineNumber, tag->language, tag->kind->letter);
+			void (* msg) (const char *const format, ...) __printf__ (1, 2) = verbose;
+			if (Option.fatalWarnings)
+				msg = notice;
+			msg ("Unexpected character (0 < *c && *c < 0x20) included in a tagEntryInfo: %s\n", base);
+			msg ("File: %s, Line: %lu, Lang: %s, Kind: %c\n",
+			     tag->inputFileName, tag->lineNumber, tag->language, tag->kind->letter);
 			if (Option.fatalWarnings)
 				error (FATAL, "Aborting");
```

Comment:
this will still abort in case of fatal-warning, so apart silencing the message by default I don't see much point?